src/clr-angular/forms/datepicker/providers/date-form-control.service.ts

```diff
@@ -49,4 +48,9 @@ export class DateFormControlService {
   public clear() {
     this.dataClearEvent.next();
   }
+
+  // friendly wrapper
+  setDisabled(state: boolean) {
```

Comment:
Why?

Reply:
xaxa - it's gone now, probably some old cache or something. Merged again with master and now is showing it correctly.